flake.nix

```diff
@@ -19,6 +19,13 @@
       repo = "nixpkgs";
```

Comment:
The formatting in this file makes it a bit difficult to read

Reply:
I haven't noticed since I haven't configured my neovim to work with nix, I would look for a way to auto format nix files tomorrow.